client/src/components/AddRoom/AddRoom.js

```diff
@@ -1,49 +1,106 @@
 import React from 'react';
 
+import axios from 'axios';
+
 import { TextField, Input, FormControlLabel, Switch, FormGroup, Button, NativeSelect,
     InputLabel, FormControl, InputAdornment, Grid, Stepper, Step, StepLabel, StepContent } from '@material-ui/core';
 import { CloudUpload, Link } from '@material-ui/icons';
+import Spinner from "../UI/Spinner/Spinner";
 
 class AddRoom extends React.Component {
     state = {
-        available: true,
-        invite: false,
-        members_limit: false,
-
         activeStep: 0,
+        roomId: 1,
```

Comment:
Could you help me pls clarify, why roomId starts with 1, not 0?

Reply:
Fixed